roles/configure/files/stack-template.yaml

```diff
@@ -1147,20 +1207,16 @@ Parameters:
     Description: Network used for the PeerVPC
   PeerConnectionID:
     Type: String
-    Description: ID of existing VPC Peering Connection that connects the PeerVPC to
-      the ClusterVPC
+    Description: ID of existing VPC Peering Connection that connects the PeerVPC to the ClusterVPC
   PublicSubnetCIDR:
     Type: String
-    Description: Subnet used for the public-facing EC2 entities; must be contained
-      within ClusterNetworkCIDR
-  ControllerSubnetCIDR:
-    Type: String
-    Description: Subnet used for the controller EC2 instances; must be contained within
-      ClusterNetworkCIDR
-  WorkerSubnetCIDR:
-    Type: String
-    Description: Subnet used for the worker EC2 instances; must be contained within
-      ClusterNetworkCIDR
+    Description: Subnet used for the public-facing EC2 entities; must be contained within ClusterNetworkCIDR
+  ControllerSubnetCIDRs:
+    Type: CommaDelimitedList
+    Description: 3 CIDR blocks used for the controller and gateway EC2 instances; must be contained within ClusterNetworkCIDR
```

Comment:
Is it worth noting that these are mapped to ControllerSubnet A-B-C respectively?